### PR TITLE
OCRAD-8:- Add location attribute type documents.

### PR DESCRIPTION
**Pull request by Akayeshmantha**

### CODE REVIEW

File: location_attribute_type.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    *uuid* | `target_location_attribute_type_uuid` | Target location attribute type resource UUID
+
+    #### Attributes
+
+      Parameter | Type | Description
+      --- | --- | ---
+      *name* | `String` | Name of the location attribute type (Required)
+      *description* | `String` | Description (Required)
+      *datatypeClassname* | `CustomDataType Resource` | Data type for the attribute type resource.OpenMRS provides **Custom data type resource** which gives flexibility to select the data type accordingly (Required)
+      *minOccurs* | `Number` | Minimum number of times this value can be specified for a single location.Use `0` or `1` as the default value (Required) (Required)
```
Comment (bmamlin) [inline location_attribute_type.md]: Only one `(Required)` is required. 🙂

File: location_attribute_type.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *name* | `String` | Name of the location attribute type (Required)
+    *description* | `String` | Description (Required)
+    *datatypeClassname* | `CustomDataType Resource` | Data type for the attribute type resource.OpenMRS provides **Custom data type resource** which gives flexibility to select the data type accordingly (Required)
+    *minOccurs* | `Number` | Minimum number of times this value can be specified for a single location.Use `0` or `1` as the default value (Required)
+    *maxOccurs* | `Number` | Maximum number of times this value can be specified for a single location (e.g., use 1 to prevent an attribute from being added to a location multiple times)
+    *preferredHandlerClassname* | `Handler` | Handler sub resource for the Custom Data Type used.Can optionally define a specific handler class want to use (otherwise the framework will choose the best handler for the chosen datatype).To find which handlers to use for the Custom DataType please refer here
```
Comment (bmamlin) [inline location_attribute_type.md]: Place space between period and beginning of next sentence.

File: location_attribute_type.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    #### Attributes
+
+      Parameter | Type | Description
+      --- | --- | ---
+      *name* | `String` | Name of the location attribute type (Required)
+      *description* | `String` | Description (Required)
+      *datatypeClassname* | `CustomDataType Resource` | Data type for the attribute type resource.OpenMRS provides **Custom data type resource** which gives flexibility to select the data type accordingly (Required)
+      *minOccurs* | `Number` | Minimum number of times this value can be specified for a single location.Use `0` or `1` as the default value (Required) (Required)
+      *maxOccurs* | `Number` | Maximum number of times this value can be specified for a single location (e.g., use 1 to prevent an attribute from being added to a location multiple times)
+      *preferredHandlerClassname* | `Handler` |  Handler sub resource for the Custom Data Type used.Can optionally define a specific handler class want to use (otherwise the framework will choose the best handler for the chosen datatype).To find which handlers to use for the Custom DataType please refer here
```
Comment (bmamlin) [inline location_attribute_type.md]: Place space between period and beginning of next sentence.

**Review by bmamlin — CHANGES_REQUESTED** [review]
Looks good. Minor suggestions.

File: location_attribute_type.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+ a `401 Unauthorized` status returned.
+
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *name* | `String` | Name of the location attribute type (Required)
+    *description* | `String` | Description (Required)
+    *datatypeClassname* | `CustomDataType Resource` | Data type for the attribute type resource. OpenMRS provides **Custom data type resource** which gives flexibility to select the data type accordingly (Required)
+    *minOccurs* | `Number` | Minimum number of times this value can be specified for a single location.Use `0` or `1` as the default value (Required)
```
Comment (bmamlin) [inline location_attribute_type.md:53]: Place a space between period and new sentence.

File: location_attribute_type.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+    *maxOccurs* | `Number` | Maximum number of times this value can be specified for a single location (e.g., use 1 to prevent an attribute from being added to a location multiple times)
+    *preferredHandlerClassname* | `Handler` | Handler sub resource for the Custom Data Type used.Can optionally define a specific handler class want to use (otherwise the framework will choose the best handler for the chosen datatype). To find which handlers to use for the Custom DataType please refer here
```
Comment (bmamlin) [inline location_attribute_type.md]: Place a space between period and new sentence.

Was "here" supposed to be a link?

**Review by bmamlin — CHANGES_REQUESTED** [review]

File: location_attribute_type.md (hunk 5 above)
Comment (Akayeshmantha) [reply location_attribute_type.md]: Hi @bmamlin  Yes it supposed to be but it's not yet done so we have to come to this when we add the document relevant.

**Comment by bmamlin** [comment]
It looks like you missed one space. I'll go ahead and merge this and then add the space.

**Pull request merged**